media-sound/denemo/denemo-2.2.0.ebuild

```diff
@@ -0,0 +1,126 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit xdg-utils
+
+DESCRIPTION="A music notation editor"
+HOMEPAGE="http://www.denemo.org/"
+SRC_URI="mirror://gnu/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-3 OFL-1.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+# configure options currently not used:
+# --enable-mem(no) memory debugging: needs Electric fence (efence), which
+#		is not available in portage. See https://github.com/boundarydevices/efence
+# --enable-gtk-doc-pdf(no) doesn't work
+IUSE="alsa +aubio coverage debug +evince doc jack +fluidsynth gtk3 nls \
+	+portaudio +portmidi +rubberband static test"
+
+RDEPEND="
+	dev-libs/libxml2:2
+	>=dev-scheme/guile-2:12
+	gnome-base/librsvg:2
+	media-libs/fontconfig:1.0
+	>=media-libs/libsmf-1.3
+	>=media-libs/libsndfile-1.0.28-r1
+	>=media-sound/lilypond-2.19.54[guile2]
+	alsa? ( >=media-libs/alsa-lib-1.1.2 )
+	aubio? ( >=media-libs/aubio-0.4.1-r1 )
+	evince? ( >=app-text/evince-3.22.1-r1 )
+	fluidsynth? ( >=media-sound/fluidsynth-1.1.6-r1 )
+	gtk3? (
+		x11-libs/gtk+:3
+		x11-libs/gtksourceview:3.0
+	)
+	!gtk3? (
+		x11-libs/gtk+:2
+		x11-libs/gtksourceview:2.0
+	)
+	jack? ( virtual/jack )
+	portaudio? (
+		>=media-libs/portaudio-19_pre20140130
+		sci-libs/fftw:3.0
+	)
+	portmidi? ( >=media-libs/portmidi-217-r1 )
+	rubberband? ( >=media-libs/rubberband-1.8.1-r1 )
+"
+
+DEPEND="${RDEPEND}
+	>=dev-util/intltool-0.51.0-r1
+	>=sys-devel/flex-2.6.1
+	virtual/pkgconfig
+	virtual/yacc
+	doc? ( >=dev-util/gtk-doc-1.25-r1 )
+	nls? ( >=sys-devel/gettext-0.19.8.1 )"
+
+DOCS=( AUTHORS ChangeLog docs/{DESIGN{,.lilypond},GOALS,TODO} NEWS )
+#HTML_DOCS=( docs/denemo-manual.html docs/denemo.css )
+
+PATCHES=( "${FILESDIR}/${P}-no-portaudio-fix.patch" )
+
+src_prepare() {
+	sed -ie '/^Categories=/s/GNOME\;/GNOME\;GTK\;/' pixmaps/denemo.desktop
+	default
+}
+
+# --enable-doc does nothing for itself
+src_configure() {
+	myeconfargs=()
+	if use doc; then
+		myeconfargs+=(
+			--enable-doc
+			--enable-gtk-doc
+			--enable-gtk-doc-html
+			--disable-gtk-doc-pdf
+		)
+	else
+		myeconfargs+=(
+			--disable-doc
+			--disable-gtk-doc
+			--disable-gtk-doc-html
+			--disable-gtk-doc-pdf
+		)
+	fi
+	econf \
+		--disable-installed-tests \
+		--disable-mem \
+		--disable-rpath \
+		--enable-guile_2_0 \
+		--enable-x11 \
+		$(use_enable alsa) \
+		$(use_enable aubio) \
+		$(use_enable coverage gcov) \
+		$(use_enable evince) \
+		$(use_enable fluidsynth) \
+		$(use_enable jack) \
+		$(use_enable nls) \
+		$(use_enable portaudio) \
+		$(use_enable portmidi) \
+		$(use_enable rubberband) \
+		$(use_enable static) \
+		$(use_enable test always-build-tests) \
```

Comment:
<del>Do you really need this? It's fine if tests are built during `src_test`.</del>

Update: hmm, if this is needed to avoid building them with `-j1`, then I guess it's fine.

Reply:
No it's not tied to the `-j1`. Tbh, I don't understand exactly what it's doing. It's somehow tied to the `installed-tests` option and also to some undocumented option `glibtest`, i.e. this option is not shown by `./configure --help` but it's in `configure.ac`. The definition is in `build/glibtests.m4` together with the option for `installed-tests`.
It compiles fine and also runs the test suite ok, if commented out, so it actually seems not needed. I was thinking about changing to to `$(use_enable test glibtest)`, but am also ok with removing it. What do you think?